### PR TITLE
Understanding babel and How it will help you Write JavaScript

### LINKED ISSUE #4110: [Languages] Understanding Babel and how will it help you write JavaScript
#### Proposed title of the article
[Languages] Understanding Babel and how will it help you write JavaScript
#### Proposed article introduction
Babel is a JavaScript transpiler that converts edge JavaScript into regular ES5 JavaScript that can run in any program (even the old ones). 

It makes accessible all the linguistic sugar that was added to JavaScript with the new ES6 determination, including classes, fat bolts, and multiline strings.
This article will get you started with Babel.
#### Key takeaways 
1. Installation
2. Classes
3. Multiline Strings
4. Fat Arrows
#### Article quality
most articles out there are not in-depth and after going through some of them it's clear that some learners may not be in a position to actually know how to apply babel and write quality javascript programs. This article will be easy and simple for understanding and executing the snippets that will be involved. 

#### References
N/A

### PR DESCRIPTION
**Pull request by Robbiewanjau**
### Article checklist
- [X] Have you used the correct folder and file structure? - **see [Contribution Guidelines](https://github.com/section-engineering-education/engineering-education/blob/master/new_contributors/CONTRIBUTING.md)**
- [X] Is your article properly formatted in Markdown? - **see [Github Markdown guide](https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet)**
- [X] Have you used our preferred heading structure? - **[H3 (###) and up](https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet#headers)**
- [X] Does your article follow any of the suggested structure formats? - **see [example formats](https://github.com/section-io/engineering-education/blob/master/CONTRIBUTING.md#example-format-structure)**

- [X] Is your article over 750 words? - **Tool**: [Wordcounter.net](https://wordcounter.net)
- [X] Is the article you are submitting an in-depth and unique article? Does it go beyond what is in the official docs and what is covered in other blog sites? See these articles as [examples](https://github.com/section-engineering-education/engineering-education/blob/master/new_contributors/resources-page.md).
- [X] Does your article provide enough value and detail about your topic? - **Articles should be clear, accurate, and fully explained.**
- [X] Can your article be understood by beginners? - **Assume the audience is smart but has no prior exposure to the common terminology in your article.**

- [X] Have you included a hero image that is 600x400 pixels and has the copyright to use it? - **Tools:** [Pixlr Image Editor](https://pixlr.com/e) for resizing and [Unsplash](https://unsplash.com) for Creative Commons images
      
#### Grammar & plagiarism checks
- [X] Have you spellchecked and grammar checked your article? - **Tools:** [Write&Improve](https://writeandimprove.com/?lang=en-GB) - [ProWritingAid](https://prowritingaid.com/) - [Grammarly](https://grammarly.com) 
- [X] Please place your article through a 3rd party plagiarism checker. We suggest using Quetext, [this tool](https://www.quetext.com/) is free and has a daily limit. **We typically accept articles with 10% or less.**
- [X] Have you checked your article for readability? - **Tool:** [Hemmingway](http://www.hemingwayapp.com)
- [ ] Have you added sources for quotes and images that aren't yours?

#### Technical checks
- [X] Are your code snippets properly formatted for syntax highlighting - **see [Syntax guide](https://gohugo.io/content-management/syntax-highlighting)**
- [X] Have you checked your code runs correctly and you've highlighted all necessary dependencies for installation?
- [X] Are the software programs and packages you're highlighting in your article up to date, using current versions and not deprecated?

#### Topic suggestion this closes
This closes #4110

### CODE REVIEW

**Review by mercymeave — CHANGES_REQUESTED** [review]
KIN

**Comment by hectorkambow** [comment]
Good afternoon and thank you for submitting your topic suggestion.
 
As a rule of thumb - please only submit articles (pull requests) with a complete and fully polished article - ready to be published. 
 
All revisions and edits should be completed on your own forked repo (as to not take up room in the queue). 
 
 
@mercymeave  **if it seems this PR is still quite a bit aways from being completed.** 

 We can close this PR to allow the author to make the required edits. 
 
Closing a PR does NOT mean the article can not be published (or fixed) it simply means that the edits, revisions, and fixes will happen outside of the queue.
 
👍 


Only if you think there are more than just a couple (2-3) edits remaining or not.

**Comment by hectorkambow** [comment]
@Robbiewanjau

File: content/articles/understanding-babel-and-how-it-will-help-you-write-javascript/index.md [hunk 1]
```diff
@@ -0,0 +1,226 @@
...
+```
+$ npm install – D babel-cli 
+```
+
+There are plugins for Webpack, Gulp, Grunt, Sublime, Webstorm, and a variety of other technologies. Babel is likely to be able to integrate with any development toolchain you may be using.
+
+### Classes
+JavaScript does not have any classes. The fact that objects inherit directly from one another means that any object can be the parent (superclass) of any other object in the system.
+
+Any function can be a constructor function, and calling it with the `new` keyword will result in the creation of a new object. You may learn more about object orientation in the JavaScript for Smart People course - Object Orientation portion, which is available here.
```
[the remark this answers is not in the document]
Comment (Robbiewanjau) [reply content/articles/understanding-babel-and-how-it-will-help-you-write-javascript/index.md]: @mercymeave, done

**Comment by Robbiewanjau** [comment]
@hectorkambow, @mercymeave, I think this PR is more than ready. Thanks

**Review by mercymeave — APPROVED** [review]
Thank you @Robbiewanjau 

In future make sure your article is properly formated and polished before opening a PR.

**Comment by hectorkambow** [comment]
jumping on this one.

**Comment by hectorkambow** [comment]
some format edits

**Pull request merged**